README.md

```diff
@@ -19,13 +19,22 @@ This project enables caching in Spring-managed beans, by using Java 5 Annotation
 
 ## Usage ##
 
+Add the repository for the Momento snapshot version:
+
+    <repositories>
+      <repository>
+        <id>sonatype-snapshots</id>
+        <url>https://s01.oss.sonatype.org/content/repositories/snapshots/</url>
+      </repository>
+    </repositories>
+
 If you are using maven, you can try it now:
 
     <dependencies>
        <dependency>
-         <groupId>com.google.code.simple-spring-memcached</groupId>
+         <groupId>software.momento.java.spring</groupId>
          <artifactId>xmemcached-provider</artifactId>
-         <version>4.1.3</version>
+         <version>4.2.14-SNAPSHOT</version>
```

Comment:
just curious how the numbers in this version bump came about; particularly the `1`->`2` for the `y` version

Reply:
I didn't change the version. I don't think we had modified the readme before, so it is showing an official version. Since we need to specify the repo I thought it would be good to make the version correct as well. I can undo the readme changes if desired, since the customer was already using a custom version from a custom repo.

Reply:
got it